### PR TITLE
documentation: Deep edit on installation/introduction.rst

### PR DESCRIPTION
**Pull request by wabernat**
Tightened language throughout.
Removed redundant :term: fields (one per term per page)
Corrected incorrect code labels.
etc.

**Component**:
Documentation

**Context**: 

**Summary**:
Many mechanical/stylistic edits to MK8s introduction. 

**Acceptance criteria**: 
Reads well, shorter, doesn't change the meaning...

---
No related ticket

### CODE REVIEW

**Comment by wabernat** [comment]
@lucieleonard gets first review! Please hold comment until she's had a chance to check up on this. There are tech pubs matters that merit discussion before we address any technical issues.

File: docs/installation/introduction.rst [hunk 1]
```diff
@@ -315,36 +317,36 @@ on a single Node, these requirements add up.
...
+of security at the port level, which must be disabled or circumvented with
+:ref:`IP-in-IP encapsulation<enable IP-in-IP>`.
 
 Also note that Kubernetes has numerous integrations with existing cloud
 providers to provide easier access to proprietary features, such as
-load balancers. For more information, see
-`this documentation article
-<https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/>`_.
+load balancers. For more information, review
+`this topic<https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/>`_.
```
Comment (lucieleonard) [inline docs/installation/introduction.rst]: this does not work and prevents the build.
I would put:
```
review Kubernetes' topic on `Cloud Providers`_.

.. _Cloud Providers: https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/
```

Comment (wabernat) [reply docs/installation/introduction.rst]: I stuck with "this topic" and fixed the bad link before I saw your review comment.

Comment (wabernat) [reply docs/installation/introduction.rst]: just fixed the bad link again...

**Comment by wabernat** [comment]
@bert-e approve

**Comment by wabernat** [comment]
@bert-e reset

**Comment by wabernat** [comment]
@bert-e approve

**Comment by alexandre-allard** [comment]
I'm closing this PR as all change set seem to have already been merged in another PR.

**Pull request closed without merging**